tools/sistr_cmd/sistr_cmd.xml

```diff
@@ -161,28 +161,28 @@
   </outputs>
   <tests>
     <test>
-      <param name="input_fastas" value="AE014613-699860.fasta"/>
+      <param name="input_fastas" value="AE014613-699860.fasta" ftype="fasta"/>
       <param name="output_format" value="tab"/>
       <output 
         name="cgmlst_profiles" 
         value="cgmlst-profiles.csv"
         ftype="csv"
         lines_diff="2">
         <assert_contents>
-          <has_text text=",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,3969539340,2545200385,225275747,2955003506,2353669245,2666669453,1672513023,3779563470,1301843222,2161147266,607954140,3680021500,2914087704,1062106200,3673111880,1314942441,1367997025,3293595301,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1528212814,2110459436,4160823845,1648892875,2084418558,1638162324,469721942,1317894045,1973458150,926214622,2197498164,398274060,,,,,,,,,,1154766063,278162969,490843778,3950769715,,,,,,4203409135,2061008354,,,,,,,1052128508,,,1510445340,,,4065472468,,,,,,1495737522,,,,,,,,3076491138,712233770,3105746335,625241463,3016847250,1928860657,2229984332,1341416065,2978539204,1175502179,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1328452594,2372254687,2640609716,3051501604,3258707132,,,,,,,,,,,,1007978530,,3646345463,1109678443,,,,,,,,,"/>
+          <has_text text=",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,3969539340,2545200385,225275747,2955003506,2353669245,2666669453,1672513023,3779563470,1301843222,2161147266,607954140,3680021500,2914087704,1062106200,3673111880,1314942441,1367997025,3293595301,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1528212814,2110459436,4160823845,1648892875,2084418558,1638162324,469721942,1317894045,1562195475,926214622,2197498164,398274060,,,,,,,,,,1154766063,278162969,490843778,3950769715,,,,,,4203409135,2061008354,,,,,,,1052128508,,,1510445340,,,4065472468,,,,,,1495737522,,,,,,,,3076491138,712233770,3105746335,2516828756,3016847250,1928860657,2229984332,1341416065,2978539204,1175502179,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,1328452594,2372254687,2640609716,3051501604,3258707132,,,,,,,,,,,,1007978530,,3646345463,1109678443,,,,,,,,,"/>
         </assert_contents>
       </output>
       <output 
         name="output_prediction_tab" 
-        value="sistr-results.tab"
+        value="sistr-results.tabular"
         ftype="tabular"
         lines_diff="2">
         <assert_contents>
           <has_text text="AE014613-699860" />
           <has_text text="Typhi" />
           <has_text text="enterica" />
           <has_text text="-:-:-" />
-          <has_n_columns n="21" />
+          <has_n_columns n="22" />
```

Comment:
Just a comment here to compare to the other output file, I think it's good to have the `<assert_contents>` section here since this output file will likely change depending on where it's run (it stores paths to files).

Reply:
If you refer to `json` output files. I also think it is better to compare them by the contents and not by similar size as it now. I had added assert text section to check invariable contents of the output file